Index: src/main/java/com/space/controller/ShipService.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/space/controller/ShipService.java	(revision 32a121586d5e47d8a4358b4fdfe9b518bbcabe38)
+++ src/main/java/com/space/controller/ShipService.java	(date 1558350575855)
@@ -3,7 +3,6 @@
 import com.space.model.Ship;
 import com.space.model.ShipType;
 import com.space.repository.ShipRepository;
-import org.hibernate.SessionFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
@@ -11,7 +10,6 @@
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.*;
 
-import javax.persistence.Entity;
 import javax.servlet.http.HttpServletRequest;
 import java.util.*;
 
